app/assets/v2/scss/gitcoin.scss

```diff
@@ -521,6 +521,7 @@
 
   .accordion-panel  {
     padding-left: 28px;
+    line-height: 36px;
```

Comment:
Can we avoid adding this here? It will make the spacing look odd on the FAQ pages, instead we could use a \<h5\> and style (locally) with:

```
    .accordion-panel h5 {
      line-height: 2.2rem;
      padding-bottom: 1.6rem;
      margin-bottom: 0px;
    }
```